### ISSUE TITLE
Cocoapods integration

### ISSUE THREAD

**Issue by CHaNGeTe** [gazedash/react-native-airplay-ios#1]
I see there is a podspec.
But project is not working with cocoapods.
Is it planned to be supported?

**Comment by gazedash** [comment]
@CHanGeTe yeah, I basically followed the examples of other libs. Do we really need the pod? Because it works fine without it, I don't see a reason do to a pod

Thanks for reaching out

**Comment by sidferreira** [comment]
```
/Users/sidferreira/Dev/Ted/TEDrn/node_modules/react-native-airplay-ios/ios/RNAirPlayManager.m:3:9: fatal error: 'RNAirplay-swift.h' file not found
#import "RNAirplay-swift.h"
```

This error happens to me when trying to use pods...

**Comment by sidferreira** [comment]
Also, `MPVolumeView.setRouteButtonImage:forState:` was deprecated in favor of `AVRoutePickerView `

**Comment by sidferreira** [comment]
@CHaNGeTe Can you check if the current version works for you?
